config/resources/domain.json

```diff
@@ -349,7 +357,7 @@
       "name": "administrative-unit",
       "class": "besluit:Bestuurseenheid",
       "super": [
-        "organization"
+        "public-organization"
```

Comment:
This public-organization is the 'Andere' right? Would it be added value to add an (extra) specific ad  hoc type to filter the out easily? Cause now, 
```
<http://data.lblod.info/id/publieke-organisaties/6852E354735BE37F15B7FAC1>
	rdf:type	<http://www.w3.org/ns/org#Organization> , <http://data.europa.eu/m8g/PublicOrganisation> .
```
looks pretty 'official' and might cause headaches if we later start processing the data. I mean, I don't think we ever want this data to flow outside the current app right?

Reply:
Is it a big deal to change it to `ver:AdHocOrganisatie`? Has only impact on resource config and migration right?